Exercise 4/check-prime.js

```diff
@@ -0,0 +1,12 @@
+function checkPrime(number) {
+  let isPrime = false;
+  for (let initial = 2; initial < number; initial++) {
+    if (number % initial == 0) {
```

Comment:
Also here I am suggesting you use variable i as it is simple and many programmers are familiar with it.